Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -589,6 +591,24 @@ extension ProjectDescription.Settings {
         if !swiftFlags.isEmpty {
             settingsDictionary["OTHER_SWIFT_FLAGS"] = .array(["$(inherited)"] + swiftFlags)
         }
+        if let cLanguageStandard = packageInfo.cLanguageStandard {
+            settingsDictionary["GCC_C_LANGUAGE_STANDARD"] = .string(cLanguageStandard)
+        }
+        if let cxxLanguageStandard = packageInfo.cxxLanguageStandard {
+            settingsDictionary["CLANG_CXX_LANGUAGE_STANDARD"] = .string(cxxLanguageStandard)
+        }
+        if let swiftLanguageVersion = packageInfo.swiftLanguageVersions?
+            .sorted(by: {
+                // If not a valid version, put it at the beginning of the sort
```

Comment:
why do we want to keep invalid versions? Shouldn't we just filter them out (possibly throw a warning or something)?